django/contrib/admin/templatetags/admin_list.py

```diff
@@ -35,36 +36,37 @@
 
 
 @register.simple_tag
-def paginator_number(cl, i):
+def paginator_number(p, i):
     """
     Generate an individual page index link in a paginated list.
     """
-    if i == cl.paginator.ELLIPSIS:
-        return format_html("{} ", cl.paginator.ELLIPSIS)
-    elif i == cl.page_num:
+    if i == p.paginator.ELLIPSIS:
+        return format_html("{} ", p.paginator.ELLIPSIS)
+    elif i == p.page_num:
         return format_html('<span class="this-page">{}</span> ', i)
     else:
         return format_html(
             '<a href="{}"{}>{}</a> ',
-            cl.get_query_string({PAGE_VAR: i}),
-            mark_safe(' class="end"' if i == cl.paginator.num_pages else ""),
+            get_query_string(p.filter_params, {PAGE_VAR: i}),
+            mark_safe(' class="end"' if i == p.paginator.num_pages else ""),
             i,
         )
 
 
-def pagination(cl):
+def pagination(p):
```

Comment:
Same as above, one-letter variable names outside of very specific use-cases isn't great